.prettierrc

```diff
@@ -2,5 +2,6 @@
     "printWidth": 100,
     "tabWidth": 4,
     "singleQuote": true,
-    "arrowParens": "always"
+    "arrowParens": "always",
+    "trailingComma": "none"
```

Comment:
my boss wants to keep the trailing comma, so let's get rid of this change, please.